app/queries/linked_items_query.rb

```diff
@@ -0,0 +1,84 @@
+class LinkedItemsQuery
+  attr_reader :content_item
+
+  def initialize(content_item)
+    @content_item = content_item
+  end
+
+  def call
+    items = linked_items
+    items["available_translations"] = available_translations if available_translations.any?
+    if content_item.format == "working_group"
+      items["policies"] = content_items_by_id.values.select { |content_item| content_item.format == "policy" }
```

Comment:
I've resolved both the issues I've described above in a rebase.

@tijmenb and I discussed his suggestion offline and our conclusion was that the frontend shouldn't know that these links are different, so using `"links"` is the right thing to do.